ui/src/papers/index.tsx

```diff
@@ -156,65 +216,92 @@ const PaperList = () => {
-                    [
-                      "1508.07252",
-                      "1706.03762",
-                      "1811.12160",
-                      "1010.2823",
-                      "1309.7440",
-                      "1404.6635",
-                    ].indexOf(paper.arxivId) !== -1
-                  );
-                })
-                .map((paper) => (
-                  <TableRow key={paper.s2Id}>
-                    <TableCell>
-                      <Typography variant="subtitle1">
-                        <strong>
-                          <a
-                            href={`https://semanticscholar.org/paper/${paper.s2Id}`}
-                          >
-                            {paper.title || paper.s2Id}
-                          </a>
-                        </strong>
-                      </Typography>
-                      {paper.authors.map((author, idx) => (
-                        <React.Fragment key={author.url}>
-                          <a href={author.url}>{author.name}</a>
-                          {idx !== paper.authors.length - 1 ? ", " : null}
-                        </React.Fragment>
-                      ))}
-                      {paper.venue && ` • ${paper.venue}`}
-                      {" •"} {paper.year}
-                    </TableCell>
-                    <TableCell>
-                      {paper.arxivId ? (
-                        <a href={`https://arxiv.org/abs/${paper.arxivId}`}>
-                          {paper.arxivId}
-                        </a>
-                      ) : null}
-                    </TableCell>
-                    <TableCell>{paper.extractedSymbolCount}</TableCell>
-                    <TableCell>{paper.extractedCitationCount}</TableCell>
-                    <TableCell>
-                      {paper.arxivId ? (
+              {papers.map((paper) => (
+                <TableRow key={paper.s2_id}>
+                  <TableCell>
+                    <Typography variant="subtitle1">
+                      <strong>
                         <a
-                          href={`/?file=https://arxiv.org/pdf/${paper.arxivId}.pdf`}
+                          href={`https://semanticscholar.org/paper/${paper.s2_id}`}
                         >
-                          Read
+                          {paper.title || paper.s2_id}
                         </a>
-                      ) : null}
-                    </TableCell>
-                  </TableRow>
-                ))}
+                      </strong>
+                    </Typography>
+                    {paper.authors.map((author, idx) => (
+                      <React.Fragment key={author.url}>
+                        <a href={author.url}>{author.name}</a>
+                        {idx !== paper.authors.length - 1 ? ", " : null}
+                      </React.Fragment>
+                    ))}
+                    {paper.venue && ` • ${paper.venue}`}
+                    {" •"} {paper.year}
```

Comment:
Is the paper.year guaranteed to not be null in the S2 API?

Reply:
I'm not sure. I don't think I changed this code, rather I just moved it. If It's `undefined` we'll just end up with a stray bullet -- a little unideal, but I *think* this page is still for internal use, so it's probably ok?  Though to your point -- when this gets baked into the UI we'll need to handle those edge cases carefully!

Reply:
For now, since we are focused on ArXiv papers, this value will always be available unless something has gone very, very wrong.

Reply:
Good point! Okay, I'm convinced this is okay to keep as is.